packages/tenant-process/test/selfcareUpsertTenant.test.ts

```diff
@@ -73,52 +74,60 @@ describe("selfcareUpsertTenant", async () => {
 
     expect(writtenPayload.tenant).toEqual(toTenantV2(updatedTenant));
   });
-  it("Should create a tenant if it does not exist", async () => {
-    const tenantSeed = {
-      externalId: {
-        origin: "Nothing",
-        value: "0",
-      },
-      name: "A tenant",
-      selfcareId: generateId(),
-      onboardedAt: mockTenant.onboardedAt!.toISOString(),
-      subUnitType: mockTenant.subUnitType,
-    };
-    const id = await tenantService.selfcareUpsertTenant(tenantSeed, {
-      authData: getMockAuthData(),
-      correlationId,
-      serviceName: "",
-      logger: genericLogger,
-    });
-    expect(id).toBeDefined();
-    const writtenEvent = await readLastTenantEvent(unsafeBrandId(id));
-    if (!writtenEvent) {
-      fail("Creation failed: tenant not found in event-store");
-    }
-    expect(writtenEvent).toMatchObject({
-      stream_id: id,
-      version: "0",
-      type: "TenantOnboarded",
-    });
-    const writtenPayload: TenantOnboardedV2 | undefined = protobufDecoder(
-      TenantOnboardedV2
-    ).parse(writtenEvent.data);
+  it.each(["IPA", "PDND_INFOCAMERE-SCP", "PDND_INFOCAMERE-PVT"])(
```

Comment:
very minor, to avoid confusion: organizations with origin `PDND_INFOCAMERE-PVT` are skipped and never created